REMARKs/AiyagariIdiosyncratic/README.md

```diff
@@ -0,0 +1,68 @@
+1. Please open this notebook in *JupyterLab*. To install it with pip, please conduct:
+
+   * pip install jupyterlab
+
+2. This notebook uses *Dolo* and *Dolark* to replicate Aiyagari(1994). To install Dolark, please follow the steps below: 
+
+   * git clone "dolo", "dolang" and "dolark" to the local directory:
+
+     ​    git clone https://github.com/econforge/dolo.git  
+     ​    git clone https://github.com/econforge/dolang.git   
+     ​    git clone https://github.com/econforge/dolark.git
+
+   *  go to the local directory and do "pip install -e . " for each of the three. (Three times in total) 
+
+     ​    pip install -e .   
+
+   * open "JupyterLab" and import dolo, dolang, and dolark. 
+
+     ​    import dolo 
+
+     ​    import dolang 
+
+     ​    import dolark 
+
+     
+
+3. This notebook also uses *altair* for plotting graphs. To install it, please do:
+
+   * pip install altair
+
+4. This notebook also uses *tabulate* to generate tables of outputs. In order to run these codes, please also install "tabulate":  
+
+   * pip install tabulate
+
```

Comment:
While executing the Jupyter notebook in the HARK env I also needed an optional pandas dependency, xlrd for reading the excel file.